Library/Homebrew/dependency_collector.rb

```diff
@@ -34,6 +34,14 @@ def initialize
     @requirements = Requirements.new
   end
 
+  def tap=(tap)
+    if (not tap.nil?) and tap.to_s =~ HOMEBREW_TAP_ARGS_REGEX
+      @tap = "#{$1}/#{$3.gsub(/^homebrew-/, "")}"
+    else
+      @tap = "Homebrew/homebrew"
+    end
+  end
```

Comment:
To be honestly, it can be strange to have the method here. Maybe put it inside `PrioritizedTapDependency`.